Tests/Web/html/tests/usersystem_tests.js

```diff
@@ -91,13 +91,20 @@ test('UserSystemTests', 'FalseAgeVerificationLoginTest', async function() {
 
     // Log in with false age verification
     {
-        await logIn(userSystem, DEFAULT_LOGIN_EMAIL, DEFAULT_LOGIN_PASSWORD, false,Services.EResultCode.Failed,Services.EResultBaseFailureReason.None);
+        await logIn(userSystem, DEFAULT_LOGIN_EMAIL, DEFAULT_LOGIN_PASSWORD, false, Services.EResultCode.Failed, Systems.ELoginStateResultFailureReason.AgeNotVerified);
+
+    }
+
+    
+    // Log in with null age verification
+    {
+        await logIn(userSystem, DEFAULT_LOGIN_EMAIL, DEFAULT_LOGIN_PASSWORD, null, Services.EResultCode.Failed, Systems.ELoginStateResultFailureReason.AgeNotVerified);
 
     }
 
     // Log in with true age verification
     {
```

Comment:
Because we are now calling `logIn` instead of CSP's `login` function, we no longer need to define a new scope here.

Reply:
sorted